GerritStats/src/main/java/com/holmsted/gerrit/processors/perperson/IdentityRecord.java

```diff
@@ -28,6 +28,7 @@
     int reviewCountPlus1;
     int reviewCountMinus1;
     int reviewCountMinus2;
+    static String rev = "";
```

Comment:
In order to save previous value when a new IdentityRecord object is instantiated. This cannot be managed if not static.